### PR TITLE
Create Milkshake recipe

### PR DESCRIPTION
**Pull request by ellikash**

### CODE REVIEW

File: drinks/Milkshake.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Milkshake recipe
+
+## Ing
```
Comment (akarhin) [inline drinks/Milkshake.md:3]:
```suggestion
## Ingredients
```

**Review by akarhin — APPROVED** [review]
Thanks!

**Review by Pinjahainen — CHANGES_REQUESTED** [review]
Nice, simple recipe. But are you sure, there's nothing else to add? Or do you need some special milk that will shake when shaked?

**Comment by ellikash** [comment]
@Pinjahainen  Thank you for noticing! Could you please look into updated recipe?

**Review by Pinjahainen — APPROVED** [review]
Nice job!

**Comment by samumantha** [comment]
Wonderful collaboration on a yummy recipe. Thank you!

**Pull request merged**